### PR TITLE
fix(TypedDefinition): Fix type definition missing error for function …

### PR DESCRIPTION
**Pull request by Seishi**
…'verifyWebpackConfig'

### CODE REVIEW

File: src/systemjs-webpack-interop.d.ts [hunk 1]
```diff
@@ -6,7 +6,9 @@ declare module "systemjs-webpack-interop" {
   export function modifyWebpackConfig(
     webpackConfig: WebpackConfig
   ): WebpackConfig;
-  export function verifyWebpackConfig(webpackConfig): void;
+  export function verifyWebpackConfig(
```
Comment (joeldenning) [inline src/systemjs-webpack-interop.d.ts]: The name of the function is `checkWebpackConfig` - please fix (even though I know you didn't add it incorrectly, it was already there)

File: src/systemjs-webpack-interop.d.ts [hunk 2]
```diff
@@ -6,7 +6,9 @@ declare module "systemjs-webpack-interop" {
...
+    webpackConfig: WebpackConfig
```
Comment (joeldenning) [inline src/systemjs-webpack-interop.d.ts]: Does this require that we depend on `@types/webpack`?

Comment (joeldenning) [reply src/systemjs-webpack-interop.d.ts]: Or actually it looks like we're already using WebpackConfig on line 7. Looks like that is defined by the `webpack` library, without the need to install `@types/webpack`. So no need to change this 😄

Comment (Seishi) [reply src/systemjs-webpack-interop.d.ts]: I’m using your plugin in .ts files. I just write `setPublicPath(...)` but it prompts the error below:

```
9:39 Parameter 'webpackConfig' implicitly has an 'any' type.
     7 |     webpackConfig: WebpackConfig
     8 |   ): WebpackConfig;
  >  9 |   export function verifyWebpackConfig(webpackConfig): void;
       |                                       ^
    10 | 
    11 |   type WebpackConfig = object | WebpackConfigGetter;
    12 |   interface WebpackConfigGetter {
```

**Comment by joeldenning** [comment]
Released in https://github.com/joeldenning/systemjs-webpack-interop/releases/tag/v1.1.1

**Pull request merged**